src/main/kotlin/no/nav/familie/ef/iverksett/brev/frittstående/DistribuerFrittståendeBrevTask.kt

```diff
@@ -63,16 +66,22 @@ class DistribuerFrittståendeBrevTask(
         }.forEach { (personIdent, journalpostResultat) ->
             try {
                 val bestillingId = distribuerBrev(journalpostResultat)
-
-                frittståendeBrev = oppdaterFrittståendeBrev(frittståendeBrev, journalpostResultat, bestillingId)
-                frittståendeBrevRepository.oppdaterDistribuerBrevResultat(
-                    frittståendeBrevId,
-                    frittståendeBrev.distribuerBrevResultat
-                )
+                frittståendeBrev = lagreEndringerPåDistribuerbrevResultat(frittståendeBrev, journalpostResultat, bestillingId, frittståendeBrevId)
             } catch (e: RessursException) {
                 val cause = e.cause
                 if (cause is HttpClientErrorException.Gone) {
                     resultat = Dødsbo("Dødsbo personIdent=$personIdent ${cause.responseBodyAsString}")
+                } else if (cause is HttpClientErrorException.Conflict) {
+                    logger.warn("Conflict: Distribuering av frittstående brev allerede utført for journalpost: ${journalpostResultat.journalpostId} ")
+                    with(objectMapper.readValue<BrevdistribusjonConflictExceptionResponseBody>(cause.responseBodyAsString)) {
+                        lagreEndringerPåDistribuerbrevResultat(
```

Comment:
Her må vel også frittståendeBrev settes? 
Litt usikker på om lesingen av `responseBodyAsString` kommer å virke som forventet, prøv å kalle, burde gå å teste å fjerne en journalpost fra resultat i databasen, og rekjøre en task som har gått ok fra før

Reply:
Eller bare logge cause.responseBodyAsString først, for å sjekke at du får dataen som du forventer.

Cause er cause på exception, se eks https://logs.adeo.no/app/discover#/doc/tjenestekall-*/tjenestekall-team-teamfamilie-prod-000021?id=d4wXGIQBfIaqcZmNSxEB

Jeg tror du nog må gjøre noe i integrasjoner for at dette skal virke :) Kanskje legge inn Brevdistribusjonskonflikt i felles kontrakter?